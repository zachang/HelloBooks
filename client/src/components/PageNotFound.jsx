import React from 'react';

/**
 * PageNotFound class declaration
 * @class PageNotFound
 * @extends {React.Component}
 */
export default class PageNotFound extends React.Component {

  /**
   * Renders PageNotFound component
   * @return {XML} JSX
   */
  render() {
    return (
      <div>
        <div className='pageNotFound'>
        </div>
      </div>
    );
  }
}