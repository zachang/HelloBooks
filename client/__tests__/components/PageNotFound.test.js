import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow, mount } from 'enzyme';

import PageNotFound from '../../src/components/PageNotFound.jsx';


describe('<PageNotFound/>', () => {
  let shallowComponent;
  beforeEach(() => {
    shallowComponent = shallow(<PageNotFound/>);
  });

  it('renders <PageNotFound/> component', () => {
    const tree = toJson(shallowComponent);
    expect(tree).toMatchSnapshot();
  });
});