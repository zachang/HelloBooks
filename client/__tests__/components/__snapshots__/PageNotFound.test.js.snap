// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound/> renders <PageNotFound/> component 1`] = `
<div>
  <div
    className="pageNotFound"
  />
</div>
`;
